server/api/event/index.js

```diff
@@ -1,17 +1,6 @@
 var router = require('express').Router();
 var db = require('../../db');
 
```

Comment:
this conflicts with derek's, we can go ahead and merge yours in first and let him rebase